hazelcast/src/main/java/com/hazelcast/jet/config/JobConfig.java

```diff
@@ -335,6 +335,7 @@ public JobConfig addClass(@Nonnull Class... classes) {
      */
     @Nonnull
     public JobConfig addPackage(@Nonnull String... packages) {
+        throwIfLocked();
```

Comment:
the exception was already thrown by `add`, often similar situation in other cases. having `throwIfLocked` in all public methods is explicit but redundant. Also technically adding empty array of packages could be allowed as it does not change state of the class.